dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -76,70 +126,267 @@ def setUpClass(cls):
+                            "skew": [1.3187609467915742, 0.8944271909999159],
+                            "kurtosis": [2.608695652173913, 1.2],
+                        },
+                    }
+                },
+            ],
+            "categorical_distribution": [
+                {
+                    "value": {
+                        "bin_counts": [1, 1, 2],
+                        "bin_edges": [4.0, 6.0, 8.0, 10.0],
+                    }
+                },
+                {
+                    "id": [
+                        {},
+                        {
+                            "bin_counts": "unchanged",
+                            "bin_edges": [[], [1.0, 2.0, 3.0, 4.0], [5.0]],
+                        },
+                        {},
+                    ]
+                },
+                {},
+            ],
+            "times": {
+                "num_nodes": "unchanged",
+                "num_edges": "unchanged",
+                "categorical_attributes": "unchanged",
+                "continuous_attributes": "unchanged",
+                "avg_node_degree": "unchanged",
+                "global_max_component_size": "unchanged",
+                "continuous_distribution": "unchanged",
+                "categorical_distribution": "unchanged",
+            },
+        }
 
     def test_profile(self):
         # test_update
-        graph_profile = GraphProfiler(self.graph)
+        graph_profile = GraphProfiler(self.graph_1)
         with utils.mock_timeit():
-            profile = graph_profile.update(self.graph)
-        scale = profile.profile["continuous_distribution"]["weight"].pop("scale")
-        continuous_distribution_props = profile.profile["continuous_distribution"][
-            "weight"
-        ].pop("properties")
-        self.assertAlmostEqual(scale, -15.250985118262854)
-        self.check_continuous_properties(continuous_distribution_props)
+            profile = graph_profile.update(self.graph_1)
         self.assertDictEqual(self.expected_profile, profile.profile)
 
     def test_report(self):
         # test_report
-        graph_profile = GraphProfiler(self.graph)
+        graph_profile = GraphProfiler(self.graph_1)
         with utils.mock_timeit():
-            profile = graph_profile.update(self.graph)
-        scale = profile.profile["continuous_distribution"]["weight"].pop("scale")
-        continuous_distribution_props = profile.profile["continuous_distribution"][
-            "weight"
-        ].pop("properties")
-        self.assertAlmostEqual(scale, -15.250985118262854)
-        self.check_continuous_properties(continuous_distribution_props)
+            profile = graph_profile.update(self.graph_1)
         self.assertDictEqual(self.expected_profile, graph_profile.report())
 
     def test_graph_data_object(self):
-        data = GraphData(input_file_path=None, data=self.graph)
+        data = GraphData(input_file_path=None, data=self.graph_1)
```

Comment:
let's remove the `input_file_path=None` and just use `data=`

Reply:
Or could we fix this to not even use `data` and directly send the self.graph_1 into the profiler?